lib/rules/no-disallowed-properties.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+var helpers = require('../helpers');
+
+module.exports = {
+  'name': 'no-disallowed-properties',
+  'defaults': {
+    'properties': []
+  },
+  'detect': function (ast, parser) {
```

Comment:
I tried to use ES6 syntax but eslint don't like it:
`10:10  error  Parsing error: Unexpected token (`